pkg/handlers/dev/configure.go

```diff
@@ -46,7 +47,7 @@ func (cmd *ConfigureCmd) Run(c *cc.CommonCtx) error {
 
 	if cmd.Name == "" && cmd.Resource == "" {
 		if cmd.LocalPolicyImage == "" {
-			return errors.New("you either need to provide a local policy image or the resource and the policy name for the configuration")
+			return errors.New("you either need to provide a local policy image, the resource and the config name or policy instance name and config name for the configuration")
```

Comment:
This `if` statement doesn't actually check the condition described in the error message.
It's possible that `cmd.Name` is not empty but `cmd.Resource`, `cmd.LocalPolicyImage`, `cmd.PolicyInstance`, and `cmd.EdgeAuthorizer` all are.
The code below uses `cmd.PolicyInstance` unconditionally but only validates that it isn't empty if `cmd.EdgeAuthorizer` is set.